.travis.yml

```diff
@@ -5,8 +5,12 @@ services:
 cache:
   directories:
   - .glide
-script:
-- make verify build test images
+jobs:
+  include:
+    - stage: Verify and Build
+      script: make verify build
+    - stage: Test and Make Image
+      script: make test images
```

Comment:
Just curious, would be make sense to put each 'make' step into its own 'stage' ?